src/Premium/Attribution.php

```diff
@@ -137,18 +146,72 @@ public function specialThanksControl( $configs ) {
 	 * @param array $controls [description]
 	 */
 	public function addControls( $controls ) {
+		global $boldgrid_theme_framework;
+		$bgtfw_version = $boldgrid_theme_framework->get_configs()['framework-version'];
+		$is_crio       = version_compare( $bgtfw_version, '2.0.0', 'ge' );
```

Comment:
Not needed, but moving this set of code to

```php
public function isCrio
```

would make it easier to reuse in the future.